src/main/resources/templates/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>hello</title>
+</head>
+<body>
+Hello world
+</body>
+</html>
```

Comment:
Add an empty line at the end of the file)

Reply:
Done